### DISCUSSION TITLE
Curso: Fullstack .Net - Identity - Login

### DISCUSSION THREAD

**Discussion by martterson** [balta-io/forum#677] [Tira Dúvidas] [answered]
Bom dia, no exemplo é utilizado as tabelas do Identity, teria algum exemplo utilizando uma tabela personalizada, pois estou tentando usar uma tabela de usuário já existente e não conseguir.
obrigado

**Comment by FBODeveloper** [comment, marked as answer]
Nesse caso você precisa criar seu próprio Endpoint ....
Tem um curso do Balta mais antigo (ele constrói um Blog) lá ele faz dessa foram.... porém, a autenticação é via token

**Reply by FBODeveloper** [reply]
Acho que é nesse curso.
[Fundamentos do ASP.NET 6](https://balta.io/player/assistir/8407be55-4137-48ae-baf5-470eef47a297)
